setup.py

```diff
@@ -0,0 +1,16 @@
+from distutils.core import setup
+
+
+# TODO: Fill this out for distribution on PyPI
+setup(
```

Comment:
You need an `install_requires` kwarg specifying dependencies; otherwise, they will not be installed. I have seen some packages read this info directly from `requirements.txt`. If you go the automatic route, remember to separate out your test-only requirements since they aren't needed for production installations.